src/lib/albertcore/src/albert/mainwindow/mainwindow.cpp

```diff
@@ -227,6 +227,12 @@ void MainWindow::setVisible(bool visible) {
 /** ***************************************************************************/
 void MainWindow::toggleVisibility() {
    setVisible(!isVisible());
+   // albert might fail to grab keyboard focus if you press keyboard immediately right after HOTKEY was triggered.
+   QTimer::singleShot(1, this, &MainWindow::regrabFocus);
```

Comment:
Well, I'm a coder. Whenever I type immediately right after I pressed the hotkey to activate Albert, the keyboard focus stayed in Editor I was typing into... By immediately I meant no break between hotkey pressing and typing..
So, I figured, it might be caused by Async Message Loop or something,  upcoming keyboard event was sent to Editor and made it keyboard focusing once again...